app/components/footer/index.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './Footer.module.css';
+import { GithubLogo, RsSchoolLogo } from '@/components/icons';
+import { GITHUB_PROJECT_URL, RSS_COURSE_URL } from '@/sources/constants.ts';
+import { footer } from '@/sources/messages/footer.ts';
+
+export const Footer = () => {
+  return (
+    <footer className={styles.container}>
+      <div className={styles.separator}></div>
```

Comment:
тут либо так
```suggestion
      <div className={styles.separator} />
```

либо вообще его снести и использовать   border-block-start: var(--border-width-sm) solid var(--color-border); для контейнера

Reply:
тоже самое что border-top